src/main/java/de/rwth/idsg/steve/web/validation/ChargeBoxIdValidator.java

```diff
@@ -28,7 +28,7 @@
  */
 public class ChargeBoxIdValidator implements ConstraintValidator<ChargeBoxId, String> {
 
-    private static final String REGEX = "\\S+";
+    private static final String REGEX = "^[a-zA-Z0-9.:_#-]+$";
```

Comment:
It's a bit restrictive. Why not accepting `Â` (I sadly have some stations with it 🥲) and others which are currently accepted?